app/uk/gov/hmrc/excisemovementcontrolsystemapi/models/messages/IE818Message.scala

```diff
@@ -29,18 +29,34 @@ case class IE818Message
   private val key: Option[String],
   private val namespace: Option[String]
 ) extends IEMessage {
-  override def localReferenceNumber: Option[String] = None
+  def localReferenceNumber: Option[String] = None
```

Comment:
Do we need a localReferenceNumber here if the message doesn't have it? could the lrnEquals method just return false?

Reply:
yes, I agree, we dont need it here. I will looke into it.

Reply:
done